engine/access/state_stream/backend/backend_events_test.go

```diff
@@ -67,16 +65,6 @@ func (s *BackendEventsSuite) TestSubscribeEventsFromLocalStorage() {
 		blockEvents[b.ID()] = events
 	}
 
-	s.events.On("ByBlockID", mock.AnythingOfType("flow.Identifier")).Return(
```

Comment:
why isn't this needed anymore? your changes shouldn't have modified the bahavior of the events endpoint